### PR TITLE
Update README, enabling technologies and protocols in docs

### PR DESCRIPTION
**Pull request by rsanchez15**

### CODE REVIEW

File: amlip_docs/rst/developer_manual/technologies/technologies.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+|amlip| framework relies on :term:`DDS` communication protocol to connect and communicate each of its :term:`Nodes <Node>`.
+:term:`DDS` protocol support :term:`publications <Publish>` and :term:`subscriptions <Subscribe>` in different :term:`Topics <Topic>` in order to create a distributed network of entities where communication takes place peer-to-peer, avoiding centralized systems and creating an homogeneous and stand-alone network.
+:term:`DDS` relies on :term:`QoS` to configure different characteristics for each of the communication channels, allowing to create really dynamic and complex networks.
+
+Fast DDS
+--------
+
+|amlip| uses |efastdds|, a C++ open-source library that implements :term:`DDS` specification.
+|efastdds| has all the features and characteristics needed to power |amlip| communications.
+A whole documentation for the |fastdds| project can be found here: |FastDDSDocs|.
```
Comment (irenebm) [inline amlip_docs/rst/developer_manual/technologies/technologies.rst:28]:
```suggestion
A whole documentation for the |fastdds| project can be found `here <|FastDDSDocs|>`_.
```

File: amlip_docs/index.rst [hunk 2]
```diff
@@ -76,6 +74,8 @@
    /rst/developer_manual/installation/sources/linux/linux
    /rst/developer_manual/installation/sources/windows/windows
    /rst/developer_manual/installation/configuration/cmake_options
+   /rst/developer_manual/technologies/technologies
```
Comment (juanlofer-eprosima) [inline amlip_docs/index.rst:77]: “The best argument against democracy is a five-minute conversation with the average voter (Winston Churchill)".

File: amlip_docs/rst/developer_manual/protocols/protocols.rst [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+All the |amlip| Topics in the DDS |amlip| network have a previous name mangling.
+So an |amlip| topic named , :code:`some_topic` would actually be named :code:`amlip::some_topic` in the underneath DDS network.
+
+.. _protocols_dds_multiservice:
+
+MultiService over DDS
+----------------------
+
+A new communication protocol based on DDS has been designed in order to fulfill the necessity of distributing a task in a network.
```
Comment (irenebm) [inline amlip_docs/rst/developer_manual/protocols/protocols.rst:20]:
```suggestion
A new communication protocol based on :term:`DDS` has been designed in order to fulfill the necessity of distributing a task in a network.
```

File: amlip_docs/rst/developer_manual/protocols/protocols.rst [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+The idea is a Service protocol based on Client-Server communication where multiple servers could be available at the same time in the same network.
+This protocol creates an auto-regulated orchestration method where a task could be distributed to **one and only one** server that is publicly available, and each server receives no more than one task at a time.
+
+.. _protocols_dds_rpc:
+
+RPC over DDS
+------------
+
+The Remote Procedure Call based on DDS has been implemented to meet the need to distribute requests and replies across a network.
```
Comment (irenebm) [inline amlip_docs/rst/developer_manual/protocols/protocols.rst:29]:
```suggestion
The Remote Procedure Call based on :term:`DDS` has been implemented to meet the need to distribute requests and replies across a network.
```

**Review by irenebm — APPROVED** [review]
LGTM

**Pull request merged**